app/templates/components/events/view/overview/general-info.hbs

```diff
@@ -13,35 +13,43 @@
       </tr>
       <tr>
         <td><strong>{{t 'Location'}}</strong></td>
-        <td>asdad</td>
+        <td>{{data.event.locationName}}</td>
       </tr>
       <tr>
         <td><strong>{{t 'Start time'}}</strong></td>
-        <td>12:30 PM May 11, 2014</td>
+        <td>{{moment-format data.event.startsAt 'MMMM Do YYYY, h:mm:ss A'}}</td>
```

Comment:
Remove the seconds from all the dates. Also the format seems to differ from the desired output.